src/main/java/org/mockbukkit/mockbukkit/block/state/BlockStateMockFactory.java

```diff
@@ -66,6 +66,7 @@ public class BlockStateMockFactory
 			.put(Material.TEST_BLOCK, with(TestBlockStateMock::new, TestBlockStateMock::new))
 			.put(Material.TEST_INSTANCE_BLOCK, with(TestInstanceBlockStateMock::new, TestInstanceBlockStateMock::new))
 			.put(Material.TRAPPED_CHEST, with(ChestStateMock::new, ChestStateMock::new))
+			.put(Material.DECORATED_POT, with(DecoratedPotStateMock::new, DecoratedPotStateMock::new))
```

Comment:
Could you please move this to be close to the dispenser so that we can keep it alphabetically sorted? Thanks!
```java
.put(Material.DAYLIGHT_DETECTOR, with(DaylightDetectorStateMock::new, DaylightDetectorStateMock::new))
.put(Material.DECORATED_POT, with(DecoratedPotStateMock::new, DecoratedPotStateMock::new))
.put(Material.DISPENSER, with(DispenserStateMock::new, DispenserStateMock::new))
```